website/docs/expansion.md

```diff
@@ -0,0 +1,223 @@
+---
+id: expansion
+title: Early Rejection of Generator Resources
+---
+
+> 🚧 This feature is in _alpha_ stage, and is not enabled by default. To
+> enable, set the "enable-generator-resource-expansion" flag.
+
+Gatekeeper can be configured to reject generator resources that might create a
```

Comment:
We should start with a definition and example of "generator resources", I don't think it's a common term.